src/standalone/Soil/energy_hydrology.jl

```diff
@@ -1167,119 +1097,35 @@ function ClimaLand.surface_specific_humidity(
     g = LP.grav(model.parameters.earth_param_set)
     R = LP.gas_constant(model.parameters.earth_param_set)
     M_w = LP.molar_mass_water(model.parameters.earth_param_set)
+    _ρ_l = LP.ρ_cloud_liq(model.parameters.earth_param_set)
+    _ρ_i = LP.ρ_cloud_ice(model.parameters.earth_param_set)
     thermo_params =
         LP.thermodynamic_parameters(model.parameters.earth_param_set)
-    ψ_sfc = p.soil.sfc_scratch
-    ClimaLand.Domains.linear_interpolation_to_surface!(
-        ψ_sfc,
-        p.soil.ψ,
-        model.domain.fields.z,
-        model.domain.fields.Δz_top,
-    )
-    q_over_ice =
-        Thermodynamics.q_vap_saturation_generic.(
-            thermo_params,
-            T_sfc,
-            ρ_sfc,
-            Thermodynamics.Ice(),
-        )
-    q_over_liq =
-        Thermodynamics.q_vap_saturation_generic.(
-            thermo_params,
-            T_sfc,
-            ρ_sfc,
-            Thermodynamics.Liquid(),
-        ) .* (@. exp(g * ψ_sfc * M_w / (R * T_sfc)))
-    ice_frac = p.soil.ice_frac
+
+    ψ_sfc = ClimaLand.Domains.top_center_to_surface(p.soil.ψ)
     ν_sfc = ClimaLand.Domains.top_center_to_surface(model.parameters.ν)
     θ_r_sfc = ClimaLand.Domains.top_center_to_surface(model.parameters.θ_r)
     S_c_sfc = ClimaLand.Domains.top_center_to_surface(
         model.parameters.hydrology_cm.S_c,
     )
     θ_l_sfc = ClimaLand.Domains.top_center_to_surface(p.soil.θ_l)
     θ_i_sfc = ClimaLand.Domains.top_center_to_surface(Y.soil.θ_i)
-    @. ice_frac = ice_fraction(θ_l_sfc, θ_i_sfc, ν_sfc, θ_r_sfc)
-    return @. (1 - ice_frac) * q_over_liq + ice_frac * q_over_ice
-end
-
-"""
-    ice_fraction(θ_l::FT, θ_i::FT, ν::FT, θ_r::FT)::FT where {FT}
-
-Computes and returns the ice fraction, which is the
-fraction of the vapor flux that is due to sublimation, and
-the fraction of the humidity in the air due to ice, as
 
-f = S_i/(S_i+S_l)
-
-This same fraction is used to estimate the specific humidity, i.e.
-q = q_over_ice * f + q_over_water * (1-f).
-"""
-function ice_fraction(θ_l::FT, θ_i::FT, ν::FT, θ_r::FT)::FT where {FT}
-    S_l = effective_saturation(ν, θ_l, θ_r)
-    S_i = effective_saturation(ν, θ_i, θ_r)
-    f = S_i / (S_i + S_l)
-    return f
-end
-
-"""
-    soil_tortuosity(θ_l::FT, θ_i::FT, ν::FT) where {FT}
-
-Computes the tortuosity of water vapor in a porous medium,
-as a function of porosity `ν` and the volumetric liquid water
-and ice contents, `θ_l` and `θ_i`.
-
-See Equation (1) of : Shokri, N., P. Lehmann, and
-D. Or (2008), Effects of hydrophobic layers on evaporation from
-porous media, Geophys. Res. Lett., 35, L19407, doi:10.1029/
-2008GL035230.
-"""
-function soil_tortuosity(θ_l::FT, θ_i::FT, ν::FT) where {FT}
```

Comment:
We had previously implement a lot of these equations, commented them out, and this adds them back to soil_hydrology_parameterizations.